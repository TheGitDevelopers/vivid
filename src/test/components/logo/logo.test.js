import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../../../components/common/logo/logo';

describe('Logo tests:', () => {
  /* Test mockups */
  const tree = renderer
    .create(<Logo />);

  /* All component tests should be listed here */
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
  it('default logo size prop is 120', () => {
    expect(tree.root.findByType(Logo).props.logoSize).toBe(120);
  });
});
