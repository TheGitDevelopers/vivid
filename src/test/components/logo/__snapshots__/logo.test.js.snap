// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo tests: renders correctly 1`] = `
<Image
  source={1}
  style={
    Array [
      Object {
        "resizeMode": "contain",
        "width": 120,
      },
      null,
    ]
  }
/>
`;
